extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/config/AstroThingConfig.java

```diff
@@ -36,6 +36,10 @@ public void parseGeoLocation() {
         if (geoParts.length == 2) {
             latitude = toDouble(geoParts[0]);
             longitude = toDouble(geoParts[1]);
+        } else if (geoParts.length == 3) {
+            latitude = toDouble(geoParts[0]);
+            longitude = toDouble(geoParts[1]);
+            altitude = Integer.parseInt(geoParts[2]);
```

Comment:
That wasn't me :-) The astro binding used `Integer` for the altitude already internaly, so I parsed that configuration value to an `Integer`. I have changed it now to `double` so its the same as lat and lon inside the binding. I am not sure whether the whole binding should make use of `BigDecimal` instead of `Double` though...